### PR TITLE
Grammar & Fixes

### PR DESCRIPTION
**Pull request by Aericio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +63,4 @@ Yes you can! Contributions are welcomed provided that they comply with our [Cont
 	You should have received a copy of the GNU Lesser General Public License
 	along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-pmmp/PocketMine and ClearSkyTeam/PocketMine is not affiliated with Mojang. All brands and trademarks belong to their respective owners. PocketMine-MP is not a Mojang-approved software, nor is it associated with Mojang.
+[pmmp/PocketMine](https://github.com/pmmp/PocketMine-MP) and [ClearSkyTeam/PocketMine](https://github.com/ClearSkyTeam/PocketMine-MP) is not affiliated with Mojang. All brands and trademarks belong to their respective owners. PocketMine-MP is not a Mojang-approved software, nor is it associated with Mojang.
```
Comment (inxomnyaa) [inline README.md]: can you make it 'is not' -> 'are not'

Comment (inxomnyaa) [reply README.md]: and PocketMine-MP is not a Mojang-approved also add ClearSky behind it please

Comment (inxomnyaa) [reply README.md]: bump @Aericio

**Review by inxomnyaa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Aericio) [reply README.md]: luuuuuuuuuuuuuul

**Pull request merged**